molecule/driver/docker.py

```diff
@@ -101,7 +101,7 @@ class Docker(base.Base):
 
     When attempting to utilize a container image with `systemd`_ as your init
     system inside the container to simulate a real machine, make sure to set
-    the ``privileged``, ``volume_mounts``, ``command``, and ``environment``
+    the ``privileged``, ``volumes``, ``command``, and ``environment``
```

Comment:
Is this dependant on some version of Docker? 

Do we need to include both or `volumes` is deprecated (or just wrong!?)?

Reply:
I was *assuming* it was just wrong, this is the only mention / example in the entire repo with `volume_mounts` every other mention / example is `volumes`. Even in this same file (lines ~66-67) it's `volumes`:

```yaml
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:ro
```